src/code/z_skelanime.c

```diff
@@ -18,7 +18,7 @@ void SkelAnime_AnimationType5Loaded(GlobalContext* globalCtx, AnimationEntryType
 
 //.data
 u32 D_8012A480 = 0;
-static AnimationEntryCallback D_8012A484[] = {
+static AnimationEntryCallback AnimationLoadDone[] = {
```

Comment:
That name doesn't fit our naming scheme, static variables should start with `s` while global variables start with `g`
```suggestion
static AnimationEntryCallback sAnimationLoadDone[] = {
```